### ISSUE TITLE
[Suggestion] Add more servers

### ISSUE THREAD

**Issue by Treadspear** [OP-Archives/xQc-site#7]
Hello, 

As we know there is a monthly limit to the bandwidth used to host the vods.

Will you consider adding an additional server? This would be great because it would help spread the usage. If you do consider this, I would defiantly check out Azure because they have the best servers for outside of the US.

Thanks

**Comment by TimIsOverpowered** [comment]
The current budget for CDN usage is set to 50TB/month. I'm sure if more people are interested, xQc would likely increase the budget. 👍

**Comment by mayeaux** [comment]
> The current budget for CDN usage is set to 50TB/month. I'm sure if more people are interested, xQc would increase the budget. 👍

Limit resets at the first of each month? We're pretty early in the month and it's already over the limit.. probably need to 5x it tbh lol. I love to watch xQC's replays while I work so I am pretty sad when the limit hits.. +1 for an increase

**Comment by TimIsOverpowered** [comment]
Don't get me wrong, I do agree it should be increased. Especially when it hit the limit early in the month. I asked xqc if he wanted to increase it. (don't get your hopes up on a reply 👻)

**Comment by mayeaux** [comment]
> Don't get me wrong, I do agree it should be increased. Especially when it hit the limit early in the month. I asked xqc if he wanted to increase it. (don't get your hopes up on a reply 👻)

Lol rather than if he wants to, I would phrase it as, 'these juicers out here are suffering we need to help these poor guys out'. But I'm just playing, thanks bro fingers crossed!

**Comment by luffyTHEKINGOFTHEPIRATES** [comment]
can any one breakdown the cost of extending the bandwith, i will see what i can do

**Comment by luffyTHEKINGOFTHEPIRATES** [comment]
or delete old vods to make more space for more recent vods?

**Comment by Treadspear** [comment]
> can any one breakdown the cost of extending the bandwith, i will see what i can do

It is at least $2500 (my math is not the best) and if they want to go beyond that they have to make a custom plan, idk if that would me more or less expensive.

ref: https://cloud.google.com/network-tiers/pricing

**Comment by Treadspear** [comment]
> It's not a storage limit but a bandwidth limit. I sent in a donation asking X if he would raise the limit and he said he would do it. 5:11:40 of the latest stream

yeah the best we can do is donate and hope he remembers. or because im a brokie, i reply under all his tweets hoping it gets likes and so he would see it

**Comment by rush9537** [comment]
> > can any one breakdown the cost of extending the bandwith, i will see what i can do
> 
> It is at least $2500 for the current plan (my math is not the best). If they want to go beyond that they have to make a custom plan, i cant say if it would be more or less expensive than $5000.
> 
> ref: https://cloud.google.com/network-tiers/pricing

That pricing is crazy, personally I'd just move to servers with an unlimited speed-capped gbps link.

For example, LeaseWeb with a 10 Gbps unmetered link is a ~$1000 addon, which gives a theoretical 3240 TB per month, of course taking in account bandwidth highs and lows, plus all the surrounding hardware like ram, disks etc.. it'd still be much cheaper.

**Comment by rush9537** [comment]
One more suggestion could also be this: https://www.backblaze.com/b2/cloud-storage-pricing.html

Not sure about the performance though!

**Comment by Treadspear** [comment]
> > > can any one breakdown the cost of extending the bandwith, i will see what i can do
> > 
> > 
> > It is at least $2500 for the current plan (my math is not the best). If they want to go beyond that they have to make a custom plan, i cant say if it would be more or less expensive than $5000.
> > ref: https://cloud.google.com/network-tiers/pricing
> 
> That pricing is crazy, personally I'd just move to servers with an unlimited speed-capped gbps link.
> 
> For example, LeaseWeb with a 10 Gbps unmetered link is a ~$1000 addon, which gives a theoretical 3240 TB per month, of course taking in account bandwidth highs and lows, plus all the surrounding hardware like ram, disks etc.. it'd still be much cheaper.

i actually got the pricing wrong i was basing it off 500TB not 50TB so its probably like $250. so actually xqc can easily upgrade it

**Comment by rush9537** [comment]
> > > > can any one breakdown the cost of extending the bandwith, i will see what i can do
> > > 
> > > 
> > > It is at least $2500 for the current plan (my math is not the best). If they want to go beyond that they have to make a custom plan, i cant say if it would be more or less expensive than $5000.
> > > ref: https://cloud.google.com/network-tiers/pricing
> > 
> > 
> > That pricing is crazy, personally I'd just move to servers with an unlimited speed-capped gbps link.
> > For example, LeaseWeb with a 10 Gbps unmetered link is a ~$1000 addon, which gives a theoretical 3240 TB per month, of course taking in account bandwidth highs and lows, plus all the surrounding hardware like ram, disks etc.. it'd still be much cheaper.
> 
> i actually got the pricing wrong i was basing it off 500TB not 50TB so its probably like $250. so actually xqc can easily upgrade it

I think you got it right, 10-150 TB/month costs $0.065 per GB delivered so: 50,000 GB (50TB) * $0.065 = $3250 per month, which is the price for the current plan if using Google Cloud.

I see a reference to BunnyCDN on the site though, so if that's the CDN then it's $0.01 per GB for EU/NA, which sets the price at $500/mo.

**Comment by llaurenss** [comment]
You could also look at Cloudflare r2, they don't seem to have any egress fees and have pricing based on data operations

**Comment by Spectroxx** [comment]
> Don't get me wrong, I do agree it should be increased. Especially when it hit the limit early in the month. I asked xqc if he wanted to increase it. (don't get your hopes up on a reply 👻)

Any update on x replying to you? @TimIsOverpowered

**Comment by TimIsOverpowered** [comment]
> > Don't get me wrong, I do agree it should be increased. Especially when it hit the limit early in the month. I asked xqc if he wanted to increase it. (don't get your hopes up on a reply 👻)
> 
> Any update on x replying to you? @TimIsOverpowered

There has not been an update.
